dir.props

```diff
@@ -25,6 +25,7 @@
   <Import Project="build/BundledSdks.props" />
   <Import Project="build/BundledTemplates.props" />
   <Import Project="build/DependencyVersions.props" />
+  <Import Condition=" Exists('$(DownloadedPackageVersionsProps)') " Project="$(DownloadedPackageVersionsProps)" />
```

Comment:
Yes, here it comes after DependencyVersion.props, so DependencyVErsions.props doesn't need conditions.

I would prefer if this import were put inside DependencyVersions.props because I suspect there will be scenarios where we need to override the flow. E.g. sometimes you need to reference a prior stable version and not the version from current orchestrated build.

Reply:
I would also prefer OrchestratedPackageVersionsProps over DownloadedPackageVersionsProps as the name.  I'd rather the name say what importing the file does rather than how the file was acquired (implementation detail).